src/auth-users/auth-users.service.ts

```
@@ -0,0 +1,19 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { CreateUser } from './services/create-user.service';
+import { UserMetaData } from 'src/utils/types/user-metadata';
+import { SignUpEmailDto } from './dto/sign-up/sign-up-email.dto';
+
+@Injectable()
+export class AuthUsersService {
+    constructor(
+        private prisma:PrismaService
+        
+    ){}
```

ESLint

[error] Replace `··constructor(⏎········private·prisma:PrismaService⏎········⏎····)` with `constructor(private·prisma:·PrismaService)·`

(prettier/prettier)